rosdep/python.yaml

```diff
@@ -1057,6 +1057,11 @@ python-oauth2:
   debian: [python-oauth2]
   fedora: [python-oauth2]
   ubuntu: [python-oauth2]
+python-oauth2client:
+  debian: [python-oauth2client]
+  fedora: [python-oauth2client]
+  ubuntu:
+    trusty: [python-oauth2client]
```

Comment:
is there a good way to find out what releases have this package other than spinning up dockers and apt-getting it?